max: 5000
cycle_duration:
  mean: 1.5
  standard_deviation: 1
waypoints:
  mean: 4
  standard_deviation: 2
  x:
    mean: 0.0
    standard_deviation: 10.0
  y:
    mean: 0.0
    standard_deviation: 15.0
features:
  size: 5
  feature0:
    probability: 0.75
    resource: 'camera'
  feature1:
    probability: 0.7
    resource: 'battery'
    type: 2
    level:
      mean: 0.6
      standard_deviation: 0.2
  feature2:
    probability: 0.6
    resource: 'processor'
    type: 1
    level:
      mean: 3
      standard_deviation: 1
  feature3:
    probability: 0.5
    resource: 'strength'
    type: 2
    level:
      mean: 5.6
      standard_deviation: 3.4
  feature4:
    probability: 0.35
    resource: 'laserscan'
